### ISSUE TITLE
Add Kiwi IRC webclient

### ISSUE THREAD

**Issue by TwistedTwigleg** [TwistedTwigleg/vanilla-bootstrap#11]
- [ ] Choose placement
- [ ] Create plugin settings
- [ ] Implement the embed
- [ ] Theme it

From #4:

> @Megalomaniak said:
> Another point of interest, current live forum has the links in it's menu to some discord channel(I don't use discord myself so no clue what channel) and a link to kiwi irc webclient to connect users to the godot irc channel.
>
> The latter allows to be embedded to sites to avoid community memebers having to follow a link to another site, basically to keep users on the community site...
https://kiwiirc.com/embedding
> 
> Might be worth investigating in future.
> 
> @TwistedTwigleg I'd be willing to take the kiwi irc as my task if you are OK with taking the discord as yours. ...

_____

@Megalomaniak, do you still want to take this issue?

**Comment by Megalomaniak** [comment]
Yep, though as I said previously, these aren't as high in terms of priorities. Good to have it assigned already though. I'm thinking I'll try implementing it as a separate plugin, though pocket might work too. Will decide when I get to it.

**Comment by TwistedTwigleg** [comment]
Alright, I just wanted to make sure and not assume.
Let me know if you need any help.
